ideascube/management/commands/tags.py

```diff
@@ -20,6 +21,13 @@ def exit(text, **kwargs):
     sys.exit(1)
 
 
+ponctuation_regex = re.compile(r"([;:.,?!+-@+-/*]+)")
```

Comment:
I'm not sure.

`Culture: musique` seems like a very valid tag to me. So does `audio-visuel`.

We might want to keep some punctuation marks?